docs/getting_started.md

```diff
@@ -0,0 +1,170 @@
+   For Linux:   KEYCLOAK_HOME/bin/standalone.sh -b 0.0.0.0
+   For Windows: KEYCLOAK_HOME\bin\standalone.bat -b 0.0.0.0
+   ````
+
+3. The URL of the Keycloak server will be http://&lt;HOSTNAME&gt;:8080 (replace &lt;HOSTNAME&gt; with the hostname of the separate host).
+
+To start the Keycloak server on different ports:
+
+1. Open a terminal and navigate to the root of the Keycloak server directory.
+
+2. The following shows the command to start the Keycloak server:
+
+   ````
+   For Linux:   KEYCLOAK_HOME/bin/standalone.sh -Djboss.socket.binding.port-offset=100
+   For Windows: KEYCLOAK_HOME\bin\standalone.bat -Djboss.socket.binding.port-offset=100
+   ````
+
+3. The URL of the Keycloak server will be *http://localhost:8180*
+
+## <a id="add-admin"></a>Add Admin User
+
+Open the main page for the Keycloak server ([localhost:8180](http://localhost:8180) or http://&lt;HOSTNAME&gt;:8080). If
+this is a new installation of Keycloak server you will be instructed to create an initial admin user. To continue with
+the quickstarts you need to do this prior to continuing.
+
+## <a id="add-roles-user"></a>Create Roles and User
+
+To be able to use the examples you need to create some roles as well as at least one sample user. To do first this open
+the Keycloak admin console ([localhost:8180/auth/admin](http://localhost:8180/auth/admin) or http://&lt;HOSTNAME&gt;:8080/auth/admin) and
+login with the admin user you created in the [add admin user](#add-admin) section.
+
+Start by creating a user role:
+
+* Select `Roles` from the menu
+* Click `Add Role`
+* Enter `user` as `Role Name`
+* Click `Save`
+
+Next create a user:
+
+* Select `Users` from the menu
+* Click `Add user`
+* Enter any values you want for the user
+* Click `Save`
+* Select `Credentials` from the tabs
+* Enter a password in `New Password` and `Password Confirmation`
+* Click on the toggle to disable `Temporary`
+* Click `Reset Password`
+* Click `Role Mappings`
+* Select `user` under `Available Roles` and click `Add selected`
+
+As an alternative to manually creating the role and user you can use the partial import feature in the admin console and import
+the file [config/partial-import.json](config/partial-import.json) into your realm.
+
+One more step, if you want to access the examples with the admin user you need to add the `user` role to admin user:
+
+* Select `Users` from the menu
+* Click `View all users`
+* Click `Edit` for admin user
+* Click `Role Mappings`
+* Select `user` under `Available Roles` and click `Add selected`
+
+## <a id="wildfly"></a>Start and Configure the <span>WildFly</span> Server
+
+Before starting the WildFly Server start by extracting the Keycloak client adapter into it.
+
+For WildFly 10 extract `keycloak-wildfly-adapter-${project.version}.zip` into WILDFLY\_HOME.
+
+If you plan to try the SAML examples you also need the SAML WildFly adapter. To do this for WildFly 10
+`keycloak-saml-wildfly-adapter-dist-${project.version}.zip` into WILDFLY\_HOME.
```

Comment:
Should we have the `${project.version}` placeholder here?

Reply:
That falls in the same situation of https://github.com/keycloak/keycloak-quickstarts/pull/151/files#r276225720. But like I said nothing stop us from improve these documents, we just need to keep the consistency across all the projects. So once it's changed in the main repo, we can change also into others.
Marking as resolved for now.